<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>919</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="StopButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>350</y>
      <width>181</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : red ; border-raduis : 5px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGraphicsView" name="Cabane">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>80</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="Bord1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>641</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="Bord2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>280</y>
      <width>641</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Bateau1">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>200</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Boat1</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="Porte1">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>160</y>
      <width>41</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="Porte2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>160</y>
      <width>41</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Bateau2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>200</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Boat2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Authentification">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>10</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Authentification</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BateauMilieu">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>200</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">style =&quot;visible:hidden&quot;</string>
    </property>
    <property name="text">
     <string>BoatMilieu</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>60</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Perso</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Vert1">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>80</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="Rouge1">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>100</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="Vert2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>80</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="Rouge2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>100</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>919</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <slots>
  <slot>on_Bateau_clicked()</slot>
 </slots>
</ui>
